_playbooks/playbook-pam.md

```diff
@@ -0,0 +1,367 @@
+2. Help desk personnel with escalated privilege to perform security-relevant processes on endpoints (e.g. install software on user laptops or change endpoint configuration settings).
+3. Managers that approve access or accounts (e.g. a manager approving access requests to a risk reporting application).
+
+Privileged users are managed as a distinct and separate identity to decrease the risk and impact to agencies mission.
+1. These employees and contractors have the ability to jeopardize sensitive information or infrastructure, whether knowingly or unknowingly.
+2. Privileged users have the potential to compromise all three core elements of information security: availability, confidentiality, and integrity.
+
+Agencies can use this playbook to help plan and implement a logical privileged user program following government-wide best practices. This playbook includes a four step process aligned with the [Federal Identity, Credential, and Access Management (ICAM) Architecture](https://playbooks.idmanagement.gov/arch/){:target="_blank"}{:rel="noopener noreferrer"} and is designed for **insider threat, ICAM, and risk management professionals interested in identifying best practices for mitigation privileged user risk** for new or existing privileged user programs. Agencies are encouraged to tailor this playbook to fit their unique organizational structure, mission needs, and requirements. Other IT program participants, including cybersecurity program managers, may find value in incorporating this playbook approach in their planning.
+
+## Key Terms
+These are key terms used throughout this document.
+- **Account Compromise** is the unauthorized use of an account including disclosure, modification, substitution, or use of sensitive information.
+- **[Insider Threat](https://csrc.nist.gov/glossary/term/insider_threat){:target="_blank"}{:rel="noopener noreferrer"}** is the potential for an insider will use her/his authorized access, wittingly or unwittingly, to do harm to the security of United States. This threat can include damage to the United States through espionage, terrorism, unauthorized disclosure of national security information, or through the loss or degradation of departmental resources or capabilities.
+- **[Functional Privileged User](https://playbooks.idmanagement.gov/docs/playbook-dira.pdf){:target="_blank"}{:rel="noopener noreferrer"}** can access information resources provided by the application and approval workflows such as approving access requests.
+- **[Privileged Account](https://csrc.nist.gov/glossary/term/privileged_account){:target="_blank"}{:rel="noopener noreferrer"}** is a system account used by a privileged user. A privileged account can belong to a single endpoint, domain, database, or application.
+- **Privileged Compromise** is either an adverse action of a privileged user or account through either an insider threat or an account compromise.
```

Comment:
I created this term for the playbook to denote either a compromise from an account (external threat) or an insider (internal threat).

Reply:
Sounds good.